Add `osu!mania Malaysia Tournament 3` wiki article (#9996)
ppy/osu-wiki: wiki/Tournaments/OMMT/3/en.md

Sunday, 9 July 2023:

| Player A |  |  | Player B | Match link |
| --: | :-: | :-: | :-- | :-- |
| **JayLye** ::{ flag=MY }:: | **5** | 0 | ::{ flag=MY }:: PurpleLune | [#1](https://osu.ppy.sh/community/matches/109407572) |
| **LloydTTG** ::{ flag=MY }:: | **5** | 0 | ::{ flag=MY }:: XxTheJKxX | [#1](https://osu.ppy.sh/community/matches/109408032) |
| **ReJust** ::{ flag=MY }:: | **5** | 0 | ::{ flag=MY }:: Kzon | [#1](https://osu.ppy.sh/community/matches/109408709) |

Monday, 10 July 2023:

| Player A |  |  | Player B | Match link |
| --: | :-: | :-: | :-- | :-- |
| **Evirir** ::{ flag=MY }:: | **5** | 1 | ::{ flag=MY }:: cirinoes | [#1](https://osu.ppy.sh/community/matches/109420495) |
| Jugemu-Jugemu ::{ flag=MY }:: | 2 | **5** | ::{ flag=MY }:: **RedCatzzButReal** | [#1](https://osu.ppy.sh/community/matches/109422830) |

### Qualifiers

Detailed statistics for this round can be found in [this spreadsheet](https://docs.google.com/spreadsheets/d/11U3G20HprnsCz5e5PUmJCwVtPBbxwGFI3DJkieeDBPE/edit) alongside the final standings on which players made it through to the next stage.

## Ruleset

> Every staff and player must respect each other and follow the [osu! community rules](/wiki/Rules), no discussion about sensitive topics (example: religion, politics) and anything that could be offensive to others. This is a friendly 1v1 4k national tournament and not the place for such discussions. If a player breaks any of the osu! community rules, the player will be punished. Tournament staff have the right to punish players that show repeated and excessive toxic behavior toward any staff member, player, or spectator either in match multiplayer chatroom, the official discord server, or the official twitch channel chatroom.

Reviewer: ```suggestion
Every staff and player must respect each other, follow the [osu! community rules](/wiki/Rules), and do not discuss sensitive topics (example: religion, politics) and anything that could be offensive to others. This is a friendly 1v1 4k national tournament and not the place for such discussions. If a player breaks any of the osu! community rules, they will be punished. Tournament staff have the right to punish players that show repeated and excessive toxic behavior toward any staff member, player, or spectator either in match multiplayer chat room, the official Discord server, or the official Twitch channel chatroom.
```

> Punishment for violating the osu! community rules include:

Reviewer: ```suggestion
Punishments for violating the osu! community rules include:
```

- Declaring the match as forfeited, or as a win by default for the other player.
- Disqualification from the entire tournament.

### Qualifiers procedure

> 1. The referee will notify the player 10-15 minutes before the lobby is scheduled to start. Once all the players are present or another 10 minutes has passed since the scheduled time, the lobby will start.
> 2. Players will play the mappool in order from Stage 1-5 twice. After the first cycle, players will be given a 5 minute break before continuing with the 2nd cycle which follows the same procedure. Players are allowed to skip this break if all players present in the lobby agrees to start the 2nd run right away.

Reviewer: ```suggestion
1. The referee will notify the player 10-15 minutes before the lobby is scheduled to start. Once all the players are present or another 10 minutes have passed since the scheduled time, the lobby will start.
2. Players will play the mappool in order from Stage 1-5 twice. After the first cycle, players will be given a 5 minute break before continuing with the 2nd cycle which follows the same procedure. Players are allowed to skip this break if all players present in the lobby agree to start the 2nd run right away.
```

### Match procedure

> 1. The referee will notify the player 10 minutes before the match is scheduled to start. Once the player is ready they must notify the referee and they will invite the player to the lobby.

Reviewer: ```suggestion
1. The referee will notify players 10 minutes before the match is scheduled to start. Once the players are ready, they must notify the referee to be invited into the lobby.
```

2. Protecting & Banning:

>    - The player will be required to roll for **protecting and banning maps** for a total of **protecting 2 maps and banning 1 map**. The player with the highest roll will be allowed to protect a map of their choice first, as well being able to ban first, this is a bit complicated so please don't hesitate to ask in the Discord server if you do not understand this explaination.

Reviewer: ```suggestion
   - Players will be required to roll for **protecting and banning maps** for a total of **protecting 2 maps and banning 1 map**. The player with the highest roll will be allowed to protect a map of their choice first, as well as being able to ban first. Don't hesitate to ask any questions related to this procedure in the Discord server.
```

>    - A player **cannot** protect 2 maps of the same type (ex: player 1 protects hb1, now player 1 cannot protect any of the remaining hybrids), this rule applies to each person individually.

Reviewer: ```suggestion
   - A player **cannot** protect 2 maps of the same type (example: if player 1 protects hb1, they can no longer protect any other remaining hybrids). This rule applies to each person individually.
```

> 3. During the protect/ban/pick phases each player will be given a 2 minute time to choose their maps, if players end up taking too long, the ref will roll a number corresponding to the number of maps in the mappool (minus tb, example the mappool has 11 maps so the ref will !roll 10), to randomly choose a map for them. This is to ensure that a match doesn't taking too long as it does not benefit anyone.

Reviewer: ```suggestion
3. During the protect/ban/pick phases each player will be given 2 minutes to choose their maps. If players end up taking too long, the referee will roll a number corresponding to the number of maps in the mappool, excluding TB (example: the mappool has 11 maps, so the referee will `!roll 10`), to randomly choose a map for them. This is to ensure that a match doesn't take too long.
```

> 4. Each player will receive one "timeout" of two minutes per match, to be used as extra time to pick a beatmap or to rest. The timeout may only be used by the player who picks the next map, the only exception being during tiebreaker scenarios, where both players are given a two minute timeout regardless.

Reviewer 2: ```suggestion
4. Each player will receive one "timeout" of two minutes per match, to be used as extra time to pick a beatmap or to rest. The timeout may only be used by the player who picks the next map, the only exception being during tiebreaker scenarios, where both players are given a two-minute timeout regardless.
```

> 4. Each player will receive one "timeout" of two minutes per match, to be used as extra time to pick a beatmap or to rest. The timeout may only be used by the player whose is picking the next map, the only exception being during tiebreaker scenarios where both players are given a two minute timeout regardless.

Reviewer: ```suggestion
4. Each player will receive one "timeout" of two minutes per match, to be used as extra time to pick a beatmap or to rest. The timeout may only be used by the player who picks the next map, the only exception being during tiebreaker scenarios, where both players are given a two minute timeout regardless.
```

> 5. Players cannot pick or use a provocative warmup to ensure that the livestream is family friendly to watch. Players are not required to have a warm up map to be played, but if they do they must send their warmups to the referee at least 5 minutes before the match begins, but the earlier the better.

Reviewer: ```suggestion
5. Players cannot pick or use provocative warm-up maps. This is to ensure that the livestream is family friendly to watch. Players are not required to play a warm-up map, but if desired, the warm-up maps must be sent to the referee at least 5 minutes before the match begins (the earlier, the better).
```

> 6. Duration of warm up maps must be under 4 minutes, but there is no star limit.

Reviewer: ```suggestion
6. Duration of warm up maps must be under 4 minutes. There is no limit on star rating.
```

### Additional procedure

1. Players can reschedule to a later qualifiers lobby if they missed their scheduled lobby.

> 2. In case of a situation where the player doesn't show on match time, the match will be delayed by 10 minutes. If the player still does not show up the winner of the match will automatically be the other player present. To avoid this please check schedules properly and do an emergency reschedule beforehand if needed but with a valid reason.

Reviewer: ```suggestion
2. In case of a situation where a player doesn't show up on time, the match will be delayed by 10 minutes. If the player still does not show up, the other player present will automatically be declared the winner of the match. To avoid this, please check schedules properly and, if necessary, do an emergency reschedule beforehand with a valid reason.
```

> 3. If a disconnection happens within 30 seconds or first 25% after the map starts, the referee has the option to abort and restart the map. Lag isn't a reason to restart the match.

Reviewer: ```suggestion
3. If a disconnection happens within 30 seconds or first 25% after the map starts, the referee has the option to abort and restart the map. A lag isn't a reason to restart the match.
```

Reviewer 2: "a lag" wouldn't be correct, but "a lag spike" would

is correct as is though

> 4. Visual mods such as HD, FL, FI and MR along with NF mod are allowed. Meanwhile EZ and HR mod are not allowed.

Reviewer: ```suggestion
4. Visual mods such as HD, FL, FI, and MR, along with NF mod, are allowed. Meanwhile, EZ and HR mod are not allowed.
```

> 5. If a player is not cooperative with the referee/intentionally stalling the match, they will be warned by the referee. If the problem persist the host has the power to end the match and decide the winner on the spot.

Reviewer: ```suggestion
5. If a player does not cooperate with the referee, or intentionally stalls the match, they will receive a warning. If the problem persists, the host has the power to end the match and decide the winner on the spot.
```

> 6. Other special and unique cases that could happen will be up to the host to decide the next moves.

Reviewer: ```suggestion
6. The host has discretion to resolve any situations not listed here as they see fit.
```

### Schedules

> The Tournament staff will be responsible for scheduling all matches and informing players of their individual schedules. If a player is not happy or comfortable with the schedule, they will have the option to request a reschedule. For more details, refer to the tournament's Discord server.

Reviewer 2: ```suggestion
The tournament staff will be responsible for scheduling all matches and informing players of their individual schedules. If a player is not happy or comfortable with the schedule, they will have the option to request a reschedule. For more details, refer to the tournament's Discord server.
```

> The Tournament staff will be responsible of scheduling all matches and informing players when they are scheduled to play. If a player is not happy or comfortable with the schedule they will have the option to request for a reschedule, details about this topic will be in the discord server.

Reviewer: ```suggestion
The Tournament staff will be responsible for scheduling all matches and informing players of their individual schedules. If a player is not happy or comfortable with the schedule, they will have the option to request a reschedule. For more details, refer to the tournament's Discord server.
```

> We will do our best to schedule each match according to each player's preferred time to play.

Reviewer: ```suggestion
The tournament staff will do their best to schedule every match according to each player's preferred time to play.
```

> - If no referee is available during match, the match shall be delayed.

Reviewer 2: ```suggestion
- If no referee is available during the match, the match shall be delayed.
```

- If no streamers are available to broadcast the match, the match will still happen without being broadcasted.

> - If no commentators are available to commentate the match but streamers are available to broadcast the match, the match will still be broadcasted but without commentary.

Reviewer: ```suggestion
- If no commentators are available to follow the match, but streamers are, the match will still be broadcasted without commentary.
```

### Extra

> 1. Please note that the staff will try their best to broadcast every match. If two matches happen at the same time, the following options are prioritized:

Reviewer 2: ```suggestion
1. Please note that the staff will try their best to broadcast every match. If two matches happen at the same time, the following options are prioritised:
```

> 1. Please note that we will try our best to broadcast every match. If two matches happen at the same time we will prioritize:

Reviewer: ```suggestion
1. Please note that the staff will try their best to broadcast every match. If two matches happen at the same time, the following options are prioritized:
```
